aten/src/ATen/cpu/vec256/vec256_double.h

```diff
@@ -136,6 +136,9 @@ template <> class Vec256<double> {
   Vec256<double> sqrt() const {
     return _mm256_sqrt_pd(values);
   }
+  Vec256<double> rsqrt() const {
+    return _mm256_div_pd(_mm256_set1_pd(1), _mm256_sqrt_pd(values));
```

Comment:
Why cannot this be `__mm256_rsqrt_pd`?

Reply:
It [doesn't exist for _pd](https://software.intel.com/sites/landingpage/IntrinsicsGuide/#text=rsqrt_&expand=4514) and the _ps version is approximate.

Reply:
Sorry about this. I was referring to [this](https://software.intel.com/en-us/cpp-compiler-18.0-developer-guide-and-reference-mm256-rsqrt-ps), hence the question. Thanks for clarifying.